specifyweb/frontend/js_src/lib/components/Header/helpers.ts

```diff
@@ -0,0 +1,10 @@
+export function formatDate(date: Date) {
```

Comment:
(optional) move this to `utils/parser/dateFormat.ts`
(optional) rename the function to `formatDateForBackEnd` or `dateToPython` or something like that (or provide a comment)
that will make this function more discoverable in case someone would need it in the future